### ISSUE TITLE
Management site databases not paging or filtering

### ISSUE THREAD

**Issue by mhk33** [cul-it/uls#1199]
From Susan Kendrick today:

Hi all,
 
The navigation on the Management Library site – at least the Databases page – seems unresponsive. When I try to change the number of databases viewed per page or to click to another page using the numbers at the bottom, nothing happens.
 
Can someone please help?  Ideally, we could just change the page so all 72 databases showed up and we didn’t have to bunch them by 10. When we moved over to WP years ago, we were told this wasn’t an option. Is that correct?

**Comment by mhk33** [comment]
Console error:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'label_page')
    at HTMLAnchorElement.<anonymous> (accessibility.js?ver=4.2.0:24:43)
    at front.min.js?ver=4.2.0:1:1666
    at Array.forEach (<anonymous>)
    at e.each (front.min.js?ver=4.2.0:1:1640)
    at accessibility.js?ver=4.2.0:19:90
    at a (front.min.js?ver=4.2.0:1:5706)
    at Object.doAction (front.min.js?ver=4.2.0:1:6237)
    at t.render (front.min.js?ver=4.2.0:1:12246)
    at t.refresh (front.min.js?ver=4.2.0:1:8584)
    at t.init (front.min.js?ver=4.2.0:1:7998)
```

**Comment by mhk33** [comment]
@jgreidy @JesiBuell79 This seems like a new bug; could it be related to the recent upstream updates for the staff profiles fix?

**Comment by mhk33** [comment]
Checking the other units for the same feature. Most are OK. But PSL is having the same issue for Chemistry and Physics:

https://physicalsciences.library.cornell.edu/chemistry-databases/
https://physicalsciences.library.cornell.edu/physics-databases/

**Comment by mhk33** [comment]
Also the Management FAQ page has the same bug: https://dev-uls-management-library-cornell-edu.pantheonsite.io/faqs/

From Susan: 

> Weird. If I start at https://johnson.library.cornell.edu/databases/?fwp_per_page=100  and then chose the number of databases to view on the page, it works and so does the numbered pages at the bottom of the screen.
It doesn’t work if I start at https://johnson.library.cornell.edu/databases/
Not sure if this helps.

**Comment by JesiBuell79** [comment]
Name: 
Kim Cornish

Email: 
ksc6@cornell.edu - also left a note on 7/19

**Comment by jgreidy** [comment]
I did some research.

FacetWP
- Asia version is 3.8.11, Management is 4.2.0
- Asia has license key, Management does not
- Asia license key expires July 26, 2023
- FacetWP sends verification code to listed email before allowing access to plugin admin stuff (license key)
	- [mlc368@cornell.edu](mailto:mlc368@cornell.edu)
	- [kek245@cornell.edu](mailto:kek245@cornell.edu)
- Management shows a message in Plugins:
	- Please install these FacetWP integration add-ons (then re-index): [Elementor](https://facetwp.com/help-center/using-facetwp-with/elementor/)

I tried adding a new pagination facet to management (after adding the Asia license key) in multidev pagin-jgr25. The sortcode was [facetwp facet="pagination"] vs [facetwp per_page="true"]. The pagination tool did not show up, so I set the shortcode back.

I guess we need to update the license for FacetWP before going much farther.

**Comment by JesiBuell79** [comment]
@mhk33 @jgreidy - I got approval from the statement of need last night and sent it to Rachel. I can let you know when it's paid for --

**Comment by mhk33** [comment]
@jgreidy Good sleuthing! Can you tell if it's the same set up for PSL? One of their pages works, but it's less than 10 results:
https://physicalsciences.library.cornell.edu/astronomy-databases/

These don't work (over 10 results):
https://physicalsciences.library.cornell.edu/chemistry-databases/
https://physicalsciences.library.cornell.edu/physics-databases/

**Comment by jgreidy** [comment]
I compared Asia (working) with Management (not working) Asia has not applied the last 5 upstream updates.

Asia works
	facetwp 3.8.11
	facetwp contitional logic 1.4.1
	eldementor 3.7.1
	elementor pro 3.7.3
	
managament not working
	facetwp 4.2.0
	FacetWP - Conditional Logic 1.4.2
	elementor 3.13.2
	elementor pro - 3.13.1
	
I downgraded these plugins to the version in the Asia site, and Management pagination started working.
https://pagin-jgr25-uls-management-library-cornell-edu.pantheonsite.io/databases/

I'm going to try another multidev for management - this time upgrading to the current versions of these plugins - maybe the plugin makers have fixed the problem(s) by now.
